account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -326,9 +343,9 @@ def update_clean_parsed_inv(self, parsed_inv):
         if parsed_inv.get('lines'):
             for line in parsed_inv['lines']:
                 line['qty'] = float_round(
-                    line['qty'], precision_digits=prec_uom)
+                    line.get('qty', 1), precision_digits=prec_uom)
                 line['price_unit'] = float_round(
-                    line['price_unit'], precision_digits=prec_pp)
+                    float(line.get('price_unit', 0)), precision_digits=prec_pp)
```

Comment:
Don't you get these from your invoice2data line templates?

Reply:
Yes, but they return as an string, and without parsing them to a float, there will be a str failure.

Reply:
I would do that kind of data manipulation in account_invoice_import_invoice2data/wizard/account_invoice_import.py

Reply:
Changed!